pkg/operation/botanist/botanist_suite_test.go

```diff
@@ -314,9 +312,9 @@ var _ = Describe("health check", func() {
 				checker                 = botanist.NewHealthChecker(map[gardencorev1alpha1.ConditionType]time.Duration{})
 			)
 
-			exitCondition, err := checker.CheckControlPlane(shoot, seedNamespace, cloudProvider, condition, deploymentLister, statefulSetLister, machineDeploymentLister)
+			_, err := checker.CheckControlPlane(shoot, seedNamespace, cloudProvider, condition, deploymentLister, statefulSetLister, machineDeploymentLister)
 			Expect(err).NotTo(HaveOccurred())
-			Expect(exitCondition).To(conditionMatcher)
+			// Expect(exitCondition).To(conditionMatcher) TODO
```

Comment:
What is left here? Can the TODO be resolved or refined to explain what is missing?

Reply:
Done